autoware_iv_external_api_adaptor/src/rtc_controller.cpp

```diff
@@ -39,12 +43,23 @@ void RTCModule::moduleCallback(const CooperateStatusArray::ConstSharedPtr messag
   module_statuses_ = message->statuses;
 }
 
+void RTCModule::autoModeCallback(const AutoModeStatus::ConstSharedPtr message)
+{
+  auto_mode_status_.module = message->module;
+  auto_mode_status_.is_auto_mode = message->is_auto_mode;
+}
+
 void RTCModule::insertMessage(std::vector<CooperateStatus> & cooperate_statuses)
 {
   cooperate_statuses.insert(
     cooperate_statuses.end(), module_statuses_.begin(), module_statuses_.end());
 }
 
+void RTCModule::insertAutoModeMessage(std::vector<AutoModeStatus> & auto_mode_status)
+{
+  auto_mode_status.insert(auto_mode_status.begin(), auto_mode_status_);
```

Comment:
It will be added to the beginning of `auto_mode_status`, is that okay?
(It may be no problem now)
If you want to insert status to the end of `auto_mode_status`, it should be
```suggestion
  auto_mode_status.insert(auto_mode_status.end(), auto_mode_status_);
```

Reply:
@rej55 I think the order is not really required in the list as it is just rtc status. So I think this change is not necessary.